controllers/email-l10n.js

```diff
@@ -3,154 +3,138 @@
 const EmailUtils = require('../email-utils')
 const AppConstants = require('../app-constants')
 const path = require('path')
-const { readdir } = require('fs/promises')
-const partialDir = path.join(path.dirname(require.main.filename), '/views/partials/email_partials')
-const { LocaleUtils } = require('./../locale-utils')
-
-let partialFilenames
-
-async function getPartialFilenames () {
-  try {
-    partialFilenames = await readdir(partialDir)
-  } catch (e) {
-    console.error(e)
-    partialFilenames = []
+const { readFile } = require('fs/promises')
+
+const partials = ['alert', 'report', 'email_verify', 'email-monthly-unresolved']
+const mockBreaches = ['Dropbox', 'Apollo', 'Adobe']
+const mockBreachStats = {
+  passwords: {
+    count: 1,
+    numResolved: 0
+  },
+  numBreaches: {
+    count: 2,
+    numResolved: 0,
+    numUnresolved: 2
+  },
+  monitoredEmails: {
+    count: 2
   }
-
-  return partialFilenames
 }
 
-async function getEmailMockUps (req, res) {
-  const email = 'example@email.com'
-  const partials = partialFilenames || await getPartialFilenames()
-
-  if (!['dev', 'heroku'].includes(AppConstants.NODE_ENV)) return notFound(req, res)
+async function getEmailMockup (req, res) {
+  if (!['dev', 'heroku', 'stage'].includes(AppConstants.NODE_ENV)) return res.sendStatus(404)
```

Comment:
Hmm... I _believe_ previously the page was accessible to all on prod (not admin-protected for anyone).  IIRC, we even had a secops bug filed for a vuln with the partial params.  (oh yeah, we did, and I added this: https://github.com/mozilla/blurts-server/commit/3d4793ffebe0d38da3547f9b36edecb3697c8b82)

These changes put the page further on lockdown with admin-only access (in particular because we added email send functionality).  Totally agreed everyone should get a 404 on prod in a perfect world.  Currently accepting non-complex suggestions if it's worth the effort! 😅

Reply:
I was going to suggest omitting the routes in prod, so everyone gets a 404, but it looks like we already do that:

https://github.com/mozilla/blurts-server/blob/cd4e437e6e1d13fe752a2c12474fb33f861e9a66/server.js#L243

So, I'm wrong, everyone gets a 404 in prod!